lib/Server.php

```diff
@@ -265,8 +265,6 @@ private function _handle($request) {
                 $error->error->code = $e->getCode();
                 $error->error->message = $e->getMessage();
 
-                throw new \Exception("Error when we tried to call '{$one->method}'", -32601);
```

Comment:
Throwing exception at this point is quite nonsense as this is the place, where exceptions thrown by the server handlers have to be processed into JSON error response.